### DISCUSSION TITLE
How can I use the date panel separately? I need to customize some of the content

### DISCUSSION THREAD

**Discussion by vela666** [element-plus/element-plus#13547] [Q&A] [answered]
In Vue2, I can directly import DateRangePanel from 'element-ui/packages/date-picker/src/panel/date-range' 

![image](https://github.com/element-plus/element-plus/assets/69613314/50247fac-c5df-42a2-8547-240975107995)

Customize it like this
![image](https://github.com/element-plus/element-plus/assets/69613314/f27d4a10-b7dc-4c7d-b156-00d136d3eeed)

**Comment by Dsaquel** [comment, marked as answer]
DatePickerPanel released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html
